From: Michael Holzheu <holzheu@linux.ibm.com>
Subject: kmsg: Update message catalog to latest IBM level (2019/03/08)
Patch-mainline: never, S/390 specific
References: bsc#1128904 LTC#176078

Description:   kmsg: Update message catalog to latest IBM level (2019/03/08)
Symptom:       Man pages in the "kernel-default-man" rpm are missing or are
               not correct.
Problem:       Source for man pages is not up-to-date because added patches
               introduced new messages or updated existing ones.
Solution:      Update messages to latest internal IBM level.
Reproduction:  1) Build kernel with selected config, e.g.:
                  $ make defconfig && make
               2) Make kmsg-doc executable:
                  $ chmod a+x scripts/kmsg-doc
               3) Check messages:
                  $ make D=1
               4) Build messages:
                  $ make D=2
               5) Check built man pages, e.g.:
                  $ man man/cpu.643eaf.9
               Note: To check a subdirectory use "make D=1 SUBDIRS=arch/s390"


Signed-off-by: Michael Holzheu <holzheu@linux.ibm.com>
Acked-by: Petr Tesarik <ptesarik@suse.com>
---
 Documentation/kmsg/IPVS             |    3 ++
 Documentation/kmsg/ism              |    2 +
 Documentation/kmsg/s390/aes_s390    |    1 
 Documentation/kmsg/s390/af_iucv     |    1 
 Documentation/kmsg/s390/appldata    |    1 
 Documentation/kmsg/s390/bpf_jit     |    1 
 Documentation/kmsg/s390/cio         |    1 
 Documentation/kmsg/s390/cpcmd       |    1 
 Documentation/kmsg/s390/cpu         |    1 
 Documentation/kmsg/s390/crc32-vx    |    1 
 Documentation/kmsg/s390/ctcm        |    1 
 Documentation/kmsg/s390/dasd        |    1 
 Documentation/kmsg/s390/dasd-eckd   |   42 ++++++++++++++++++++++++++++++++++++
 Documentation/kmsg/s390/diag288_wdt |    1 
 Documentation/kmsg/s390/extmem      |    1 
 Documentation/kmsg/s390/hmcdrv      |    1 
 Documentation/kmsg/s390/hugetlb     |    1 
 Documentation/kmsg/s390/hypfs       |    1 
 Documentation/kmsg/s390/lcs         |    1 
 Documentation/kmsg/s390/monwriter   |    1 
 Documentation/kmsg/s390/netiucv     |    1 
 Documentation/kmsg/s390/paes_s390   |    3 +-
 Documentation/kmsg/s390/perf        |    1 
 Documentation/kmsg/s390/qeth        |    1 
 Documentation/kmsg/s390/setup       |    1 
 Documentation/kmsg/s390/vmur        |    1 
 Documentation/kmsg/s390/zfcp        |    2 +
 Documentation/kmsg/s390/zpci        |    1 
 Documentation/kmsg/s390/zram        |    2 +
 Documentation/kmsg/sbp_target       |    3 +-
 Documentation/kmsg/smc              |    1 
 31 files changed, 79 insertions(+), 2 deletions(-)

--- a/Documentation/kmsg/IPVS
+++ b/Documentation/kmsg/IPVS
@@ -79,3 +79,6 @@
 /*? Text: "%s selects TX queue %d, but real number of TX queues is %d\n" */
 /*? Text: "Unknown mcast interface: %s\n" */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
+/*? Text: "%s(): request for already hashed, called from %pS\n" */
+/*? Text: "%s(): request for unhash flagged, called from %pS\n" */
--- /dev/null
+++ b/Documentation/kmsg/ism
@@ -0,0 +1,2 @@
+/*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/aes_s390
+++ b/Documentation/kmsg/s390/aes_s390
@@ -43,3 +43,4 @@
  * None.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/af_iucv
+++ b/Documentation/kmsg/s390/af_iucv
@@ -21,3 +21,4 @@
 /*? Text: "flen=%u proglen=%u pass=%u image=%pK from=%s pid=%d\n" */
 /*? Text: "%s selects TX queue %d, but real number of TX queues is %d\n" */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/appldata
+++ b/Documentation/kmsg/s390/appldata
@@ -89,3 +89,4 @@
 /*? Text: "netif_stop_queue() cannot be called before register_netdev()\n" */
 /*? Text: "%s selects TX queue %d, but real number of TX queues is %d\n" */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/bpf_jit
+++ b/Documentation/kmsg/s390/bpf_jit
@@ -14,3 +14,4 @@
  * Report this problem and the error message to your support organization.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/cio
+++ b/Documentation/kmsg/s390/cio
@@ -245,3 +245,4 @@
  * Report the problem to your support organization.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/cpcmd
+++ b/Documentation/kmsg/s390/cpcmd
@@ -15,3 +15,4 @@
  * machine.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/cpu
+++ b/Documentation/kmsg/s390/cpu
@@ -52,3 +52,4 @@
  * does not include all CPU information.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/crc32-vx
+++ b/Documentation/kmsg/s390/crc32-vx
@@ -1 +1,2 @@
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/ctcm
+++ b/Documentation/kmsg/s390/ctcm
@@ -200,3 +200,4 @@
 /*? Text: "flen=%u proglen=%u pass=%u image=%pK from=%s pid=%d\n" */
 /*? Text: "%s selects TX queue %d, but real number of TX queues is %d\n" */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/dasd
+++ b/Documentation/kmsg/s390/dasd
@@ -691,3 +691,4 @@
  * or set the sysfs 'use_diag' attribute of the DASD to 0 to switch off DIAG.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/dasd-eckd
+++ b/Documentation/kmsg/s390/dasd-eckd
@@ -2163,4 +2163,46 @@
  * to the path_reset sysfs attribute of the device.
  * If the problem persists, report it to your support organization.
  */
+
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+
+/*?
+ * Text: "%s: Path %x.%02x (pathmask %02x) is operational despite excessive IFCCs\n"
+ * Severity: Error
+ * Parameter:
+ *   @1: bus ID of the DASD
+ *   @2: cssid
+ *   @3: chpid
+ *   @4: logical path mask
+ * Description:
+ * The threshold value for interface or channel control checks (IFCCs) for the
+ * channel path was exceeded.
+ * The channel path remains operational because the autodisable feature for
+ * defective channel paths is not active.
+ * Defective channel paths can adversely affect performance.
+ * User action:
+ * Ensure that the cabling between the storage server and the mainframe
+ * system is securely in place.
+ * If you observe performance impacts, consider setting the channel path offline.
+ * By default, the DASD device driver disables defective channel paths for you.
+ * You can restore this default, for example, by writing '1' to the
+ * path_autodisable sysfs attribute.
+ * If the problem persists, report it to your support organization.
+ */
+
+/*?
+ * Text: "%s: Last path %x.%02x (pathmask %02x) is operational despite excessive IFCCs\n"
+ * Severity: Error
+ * Parameter:
+ *   @1: bus ID of the DASD
+ *   @2: cssid
+ *   @3: chpid
+ *   @4: logical path mask
+ * Description:
+ * The threshold value for interface or channel control checks (IFCCs) for the channel path was exceeded.
+ * The channel path was not removed from regular operations because it is the last remaining channel path for the DASD.
+ * User action:
+ * Ensure that the cabling between the storage server and the mainframe
+ * system is securely in place.
+ * If the problem persists, report it to your support organization.
+ */
--- a/Documentation/kmsg/s390/diag288_wdt
+++ b/Documentation/kmsg/s390/diag288_wdt
@@ -64,3 +64,4 @@
  * contact your support organization.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/extmem
+++ b/Documentation/kmsg/s390/extmem
@@ -292,3 +292,4 @@
  * Ensure that the DCSS range is defined below the kernel mapping range.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/hmcdrv
+++ b/Documentation/kmsg/s390/hmcdrv
@@ -20,3 +20,4 @@
  * the cache size specification.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/hugetlb
+++ b/Documentation/kmsg/s390/hugetlb
@@ -11,3 +11,4 @@
  * Specify "2G" for 2 GB huge pages. These are supported as of zEC12
  * and zBC12 machines.
  */
+/*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
--- a/Documentation/kmsg/s390/hypfs
+++ b/Documentation/kmsg/s390/hypfs
@@ -55,3 +55,4 @@
 
 /*? Text: "Hypervisor filesystem mounted\n" */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/lcs
+++ b/Documentation/kmsg/s390/lcs
@@ -167,3 +167,4 @@
 /*? Text: "flen=%u proglen=%u pass=%u image=%pK from=%s pid=%d\n" */
 /*? Text: "%s selects TX queue %d, but real number of TX queues is %d\n" */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/monwriter
+++ b/Documentation/kmsg/s390/monwriter
@@ -15,3 +15,4 @@
  * in "z/VM CP Programming Services".
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/netiucv
+++ b/Documentation/kmsg/s390/netiucv
@@ -154,3 +154,4 @@
 /*? Text: "flen=%u proglen=%u pass=%u image=%pK from=%s pid=%d\n" */
 /*? Text: "%s selects TX queue %d, but real number of TX queues is %d\n" */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/paes_s390
+++ b/Documentation/kmsg/s390/paes_s390
@@ -1 +1,2 @@
-/*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
\ No newline at end of file
+/*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/perf
+++ b/Documentation/kmsg/s390/perf
@@ -88,3 +88,4 @@
  * process has ended.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/qeth
+++ b/Documentation/kmsg/s390/qeth
@@ -966,3 +966,4 @@
  * sysfs attribute of the device.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/setup
+++ b/Documentation/kmsg/s390/setup
@@ -163,3 +163,4 @@
  * None.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/vmur
+++ b/Documentation/kmsg/s390/vmur
@@ -46,3 +46,4 @@
 /*? Text: "%s loaded.\n" */
 /*? Text: "%s unloaded.\n" */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/s390/zfcp
+++ b/Documentation/kmsg/s390/zfcp
@@ -707,3 +707,5 @@
  * support organization of the storage system.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
+/*? Text: "Invalid address limit on user-mode return" */
--- a/Documentation/kmsg/s390/zpci
+++ b/Documentation/kmsg/s390/zpci
@@ -40,3 +40,4 @@
  * If the problem persists, contact your support organization.
  */
 /*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- /dev/null
+++ b/Documentation/kmsg/s390/zram
@@ -0,0 +1,2 @@
+/*? Text: "Can't setup backing device for initialized device\n" */
+/*? Text: "setup backing device %s\n" */
--- a/Documentation/kmsg/sbp_target
+++ b/Documentation/kmsg/sbp_target
@@ -46,4 +46,5 @@
 /*? Text: "unlink LUN: failed to update unit directory\n" */
 /*? Text: "flen=%u proglen=%u pass=%u image=%pK from=%s pid=%d\n" */
 /*? Text: "%s selects TX queue %d, but real number of TX queues is %d\n" */
-/*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
\ No newline at end of file
+/*? Text: "%s: %d output lines suppressed due to ratelimiting\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
--- a/Documentation/kmsg/smc
+++ b/Documentation/kmsg/smc
@@ -6,3 +6,4 @@
 /*? Text: "%s selects TX queue %d, but real number of TX queues is %d\n" */
 /*? Text: "%s: sock_register fails with %d\n" */
 /*? Text: "%s: ib_register fails with %d\n" */
+/*? Text: "%s: SME is active, device will require DMA bounce buffers\n" */
